_pages/pages/federalist-migration.md

```diff
@@ -47,6 +47,52 @@ title: cloud.gov Pages - Federalist Migration
   </div>
 </section>
 
+<section class="usa-section">
+  <div class="grid-row">
+    <div class="tablet:grid-col-7 usa-prose">
+      <h2>Migration Process</h2>
+    </div>
+  </div>
+  <div class="grid-row">
+    <div class="grid-row margin-top-4">
+      <div class="tablet:grid-col-12 usa-prose">
+        <p>
+          The primary task of migration is changing the way you authenicate. Instead of using your Github account, you will authenticate using cloud.gov’s authentication service. In some cases, this will allow you to leverage your agency’s existing identity infrastructure, otherwise, you will use an identity service provided by cloud.gov.
```

Comment:
```suggestion
          The primary task of migration is changing the way you authenticate. Instead of using your Github account, you will authenticate using cloud.gov’s authentication service. In some cases, this will allow you to leverage your agency’s existing identity infrastructure, otherwise, you will use an identity service provided by cloud.gov.
```